extensions/gdx-setup/src/com/badlogic/gdx/setup/GdxSetupUI.java

```diff
@@ -493,6 +493,12 @@ private void uiStyle() {
 				extensionPanel.setOpaque(true);
 				extensionPanel.setBackground(new Color(46, 46, 46));
 			}
+
+			nameLabel.setToolTipText("The name of the application used in gradle");
+			packageLabel.setToolTipText("The package name of the application");
+			gameClassLabel.setToolTipText("The name of the main class implementing ApplicationListener");
+			destinationLabel.setToolTipText("The root directory of the project, it will be created if it does not exist");
```

Comment:
I simplified it a bit. The problem with "Where this will generate the project's root directory" (with me) is that suggests that the root directory will be generated, but what in fact it will be filled with files. So you need to set `path/to/projects/gamename` and not `path/to/projects/`

Reply:
Looks good.